Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM jfloff/alpine-python:3.6
+
+COPY . /tmp/source
+
+WORKDIR /tmp/source
+
+RUN wget ftp://ftp.acousticbrainz.org/pub/acousticbrainz/essentia-extractor-v2.1_beta2-linux-x86_64.tar.gz
```

Comment:
because docker caches each RUN line, we should do this all in one -
```
RUN wget .... && tar xf ... && cp .... && rm -rf /tmp/source
```

Because docker invalidates caches when files change, you should put the `COPY . /tmp/source` line _after_ the wget, because it potentially changes more often than extractor file.

Reply:
:thinking: I can't put everything on the same line and copy the source afterwards because `python setup.py install` needs `streaming_extractor_music`, which it doesn't find in /usr/bin during that command. I'm drafting another version